docs/pages/config-plugins/plugins-and-mods.mdx

```diff
@@ -348,7 +348,7 @@ If an **app.plugin.js** file is present in the root of a Node module's folder, i
 - <NoIcon /> `'expo-splash-screen/app.plugin.js'`
 
 ```
-╭── app.config.js ➡️ Expo Config
+╭── app.config.js ➡️ app config
```

Comment:
```suggestion
╭── app.config.js ➡️ App config
```

All the other comments are written as a sentence, so I feel like capitalizing first word fits better here. this also applies to other replacement on in this file.